pages/common/git-rev-parse.md

```diff
@@ -0,0 +1,12 @@
+# git rev-parse
+
+> Parses a revision.
+> More information: <https://git-scm.com/docs/git-rev-parse>.
+
+- Get the absolute path to the root directory:
```

Comment:
Not sure this should be the first example since the root directory doesn't typically change between revisions. WDYT about moving it to the end of the page?

Reply:
Done in 3923a26b0f1ffe04e37f7fb476d1add4d92bdf87